dm/worker/status.go

```diff
@@ -50,25 +50,27 @@ func (w *Worker) Status(stName string) []*pb.SubTaskStatus {
 	w.RLock()
```

Comment:
can we remove them now?

Reply:
exclusive with `Close`.

Reply:
removed in https://github.com/pingcap/dm/pull/259/commits/01f693ba86374689365439155952183820910bb5.